### ISSUE TITLE
Adjusting GestureDetector hitbox

### ISSUE THREAD

**Issue by lukepighetti** [flutter/flutter#18553]
Hello,

I tried to use Transform to adjust GestureDetector's hitbox. I even tried to use two transforms, one with `transformHitTests = true` and one with `transformHitTest = false` in an attempt to move the widget and transforms down, then move the widget only up. That failed.

So I haven't found a good way to adjust a hitbox, something that is very easy in react-native using the `hitSlop` property.

Any insights?

<img width="800" alt="screen shot 2018-06-17 at 8 56 34 pm" src="https://user-images.githubusercontent.com/666539/41513966-faa2c110-7270-11e8-8705-e9f1170af5fe.png">

**Comment by zoechi** [comment]
follow-up to #14794

**Comment by lukepighetti** [comment]
Wow, I opened up two of these? I must have been really tired last night. Sorry about that.

**Comment by zoechi** [comment]
@lukepighetti thanks for the pointer

closing as dup of #18554
